Phonebook.Frontend/src/app/shared/states/App.state.ts

```diff
@@ -36,10 +47,14 @@ export interface AppStateModel {
     serviceWorkerNotificationDisplayed: false,
     version: '0.0.0',
     displayedNotificationVersion: 0,
-    sendFeedback: null
+    sendFeedback: null,
+    activeTheme: Theme.magenta_light_theme
```

Comment:
can we maybe add a `Theme.default`? So we mustn't remember the defaultValue at 100 points in the code? In this PR I have seen this line 3 times I think. So it would be a good improvement.

Reply:
just edited the code so that it is used only once now